### PR TITLE
Adds layout/chrome and footers (SUL & global) plus fixes for rubocop and scss-lint

### LINKED ISSUE #5: Add site layout/chrome
screen shot 2015-04-01 at 2 08 34 pm

### LINKED ISSUE #7: Add global footer & SUL footer
Similar to SearchWorks

screen shot 2015-04-01 at 2 09 40 pm

### PR DESCRIPTION
**Pull request by jchristo4**
Closes #5 
Closes #7

image

@jkeck Closed old PR and created this to test hound notifications

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -6,3 +6,9 @@ AllCops:
     - 'config/**/*'
     - 'spec/spec_helper.rb'
     - 'vendor/**/*'
+
+Metrics/LineLength:
+  Max: 120
+
+Style/StringLiterals:
+  EnforcedStyle: single_quotes
```
Comment (jchristo4) [inline .rubocop.yml]: @jkeck Looks like hound is ignoring this config

File: .scss-style.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    enabled: true
+    max_depth: 5
+
+  SelectorDepth:
+    enabled: true
+    max_depth: 5
+
+  StringQuotes:
+    enabled: true
+    style: single_quotes
```
Comment (jchristo4) [inline .scss-style.yml]: Added additional `single_quotes` config for scss files

File: .rubocop.yml (hunk 1 above)
Comment (jkeck) [inline .rubocop.yml]: @jchristo4 this only applies to ruby style enforcement.  We need to add the following to `.scss-style.yml` to affect the enforcement of scss rules.

      StringQuotes:
        style: single_quotes

Comment (jkeck) [inline .rubocop.yml]: (may need `enabled: true` in there, not sure)

Comment (jchristo4) [inline .rubocop.yml]: @jkeck done

File: app/views/shared/_top_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <header id="topnav" class="header-logo" role="banner">
+    <%= link_to 'http://library.stanford.edu' do %>
+      <%= content_tag :span, "Stanford University Libraries", class: "sr-only" %>
+      <%= image_tag "sul-logo.png", class: "hidden-xs", alt: "", height: 25 %>
+      <%= image_tag "sul-logo-small.png", class: "visible-xs", alt: "", height: 25 %>
+    <% end %>
+
+    <div class="header-links">
+      <%= link_to "My Account", "http://library.stanford.edu/myaccount", class: 'first' %>
+      <%= link_to "Feedback", "#"  %>
```
Comment (jkeck) [inline app/views/shared/_top_navbar.html.erb]: This should link to http://searchworks.stanford.edu/feedback per https://github.com/sul-dlss/sul-requests/pull/23/files#r27709022

Comment (jchristo4) [inline app/views/shared/_top_navbar.html.erb]: @jkeck Added SW feedback link to top nav bar

**Pull request merged**